---
string current_team
int64 remaining_reports
float64 run_clock
int64 score